tests/test.cpp

```diff
@@ -4324,6 +4324,78 @@ void FlatbuffersIteratorsTest() {
 void FlatbuffersIteratorsTest() {}
 #endif
 
+void PrivateAnnotationsLeaks() {
+  // Simple schemas and a "has optional scalar" sentinal.
+  std::vector<std::string> schemas;
+  schemas.push_back(
+      "table Monster (private) { mana : int; } struct ABC (private) { mana: "
+      "int; }");
```

Comment:
nit: it'd be a bit more readable if there was one definition per line, though no need to fight the formatter if it doesn't like that